src/hooks/useUploadReferral.ts

```diff
@@ -29,8 +32,13 @@ const useUploadReferral = () => {
     };
 
     const handleOnClick = (href: string) => {
-        console.log(href);
-        navigate(href);
+        if(usersMock[0].recipientUserId) {
```

Comment:
I believe this is done to be quick but getting the user this way? I mean, where would you get their data from if there was actually a user? 

Would it be stored in a redux store or do you get it from a service, would it be passed through a prop?